pkg/cnab/extensions/testdata/bundle.json

```diff
@@ -42,9 +42,9 @@
       "env": "equux"
     }
   },
-  "requiredExtensions": {
-    "dependencies": "https://cnab.io/specs/v1/dependencies.schema.json"
-  },
+  "requiredExtensions": [
+    "https://cnab.io/specs/v1/dependencies.schema.json"
```

Comment:
Looks like required extensions changed when I wasn't around? This isn't something you need to change. I'll follow up on it

https://github.com/deislabs/cnab-spec/blob/master/101-bundle-json.md#required-extensions

```json
"requiredExtensions":[ 
      "io.cnab.dependencies"
   ],
```